### PR TITLE
fixes #25000 - fix remote execution template sync

### PR DESCRIPTION
**Pull request by stbenjam**
I fixed the script and resynced templates.

### CODE REVIEW

File: app/views/templates/ssh/module_action.erb [hunk 1]
```diff
@@ -4,7 +4,7 @@ name: Module Action - SSH Default
 model: JobTemplate
 job_category: Modules
 description_format: "Module %{action} %{module_spec}"
-provider_type: SSH
+provider_type: Ansible
```
Comment (stbenjam) [inline app/views/templates/ssh/module_action.erb]: @parthaa This is synced from community templates - looks like this change snuck in some how, can you update the community-template and I'll resync?

Comment (iNecas) [reply app/views/templates/ssh/module_action.erb]: Great job @stbenjam! I would merge already if there was not this issue.

Comment (parthaa) [reply app/views/templates/ssh/module_action.erb]: @stbenjam thanks!

https://github.com/theforeman/community-templates/pull/518

**Comment by stbenjam** [comment]
Updated

**Comment by iNecas** [comment]
Test failures unrelated. Thanks @stbenjam and congrats to the #25000! :)

**Pull request merged**